model/cluster_request.go

```diff
@@ -181,16 +188,20 @@ func NewUpdateClusterRequestFromReader(reader io.Reader) (*UpdateClusterRequest,
 
 // PatchUpgradeClusterRequest specifies the parameters for upgrading a cluster.
 type PatchUpgradeClusterRequest struct {
-	Version       *string        `json:"version,omitempty"`
-	KopsAMI       *string        `json:"kops-ami,omitempty"`
-	RotatorConfig *RotatorConfig `json:"rotatorConfig,omitempty"`
+	Version        *string        `json:"version,omitempty"`
+	KopsAMI        *string        `json:"kops-ami,omitempty"`
+	MaxPodsPerNode *int64         `json:"max-pods-per-node,omitempty"`
```

Comment:
nit: I think in most places we use camel case, so unless there are some specific reasons against it I would stick to that.